mmv1/products/compute/terraform.yaml

```diff
@@ -1441,7 +1441,13 @@ overrides: !ruby/object:Overrides::ResourceOverrides
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       encoder: 'templates/terraform/encoders/normalize_group.go.erb'
   InstanceTemplate: !ruby/object:Overrides::Terraform::ResourceOverride
-    exclude: true
+    examples:
```

Comment:
For some reason, `exclude: true` is needed here and the tests should be added to this fie https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/tests/resource_compute_instance_template_test.go.erb

Otherwise, without `exclude: true` here, resource_compute_instance_template_sweeper_test.go will be generated automatically. However, there is already the handwritten sweeper file resource_compute_instance_template_sweeper_test.go      https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/tests/resource_compute_instance_template_sweeper_test.go. That is the reason for the failed generate-diffs check.

Reply:
Many thanks Zhenhua. You saved me hours of debugging.

Reply:
You are welcome.